src/js/components/charts/ResourceChart.js

```diff
@@ -1,102 +0,0 @@
-import PropTypes from "prop-types";
```

Comment:
could you please research dangling styles?

Reply:
Looks good I think